FutureNorthants/northants-website
# Feature/frontend video component

This adds a video component to the frontend website. It depends on the following PR's:

- https://github.com/FutureNorthants/northants-design-system/pull/85
- https://github.com/FutureNorthants/northants-website/pull/202

## To Test

- Update the backend with the above PR, edit a service page and add a video slice to a page
- Update the design-system with the above PR, then `yalc publish`
- Checkout this branch on the frontend, then `yalc add northants-design-system`, then `yarn`, then `yarn dev`
- In a new terminal window run `yarn codegen` 
- Visit the page you edited
- If you have accepted cookies you should see an embedded video
- If you have not accepted cookies you should see a text link with the description text 

## Once approved
Once all PR's are approved, the design system needs to be pushed to npm and then this branch's `yarn.lock` updated to use the latest version of the design system.
